.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+{
+  "version": "0.2.0",
+  "sourceRoot": "${workspaceRoot}/app",
+  "sourceMaps": true,
+  "sourceMapPathOverrides": {
+    // if you override this, you MUST provide all defaults again
```

Comment:
I should probably have linked to https://gist.github.com/jarshwah/389f93f2282a165563990ed60f2b6d6c ? for this